File: app/src/main/java/org/mifos/mobile/ui/views/HelpView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.models.FAQ;
import org.mifos.mobile.ui.views.base.MVPView;

import java.util.ArrayList;

/**
 * Created by dilpreet on 12/8/17.
 */

public interface HelpView extends MVPView {

    void showFaq(ArrayList<FAQ> faqArrayList);

}

File: app/src/main/java/org/mifos/mobile/ui/views/HelpView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.FAQ
import org.mifos.mobile.ui.views.base.MVPView
import java.util.*

/**
 * Created by dilpreet on 12/8/17.
 */
interface HelpView : MVPView {
    fun showFaq(faqArrayList: ArrayList<FAQ?>?)
}